gax-grpc/BUILD.bazel

```diff
@@ -2,9 +2,9 @@ load("//:gax_java.bzl", "google_java_format", "google_java_format_verification",
 
 _JAVA_COPTS = [
     "-source",
-    "1.7",
+    "1.8",
```

Comment:
I think it can be just 8, but good enough (1.8 and 8 are synnonyms, but there is no 1.9 and above, just 9, 10, 11...)